build/commands/lib/config.js

```diff
@@ -273,6 +273,10 @@ Config.prototype.buildArgs = function () {
     dcheck_always_on:
       getNPMConfig(['dcheck_always_on'], this.isComponentBuild()),
     brave_channel: this.channel,
+    // Limit action pool (non-compile actions) to amount of CPU cores.
```

Comment:
I guess I didn't look at this closely enough because we're setting this to the default which is pointless because changing it does not appear to impact rebuilds

Reply:
huh. you're right. it's a noop :/
I guess I was bamboozled by `use_goma` appearance in the gni file which I quickly interpret as "limit action pool when goma enabled, don't limit otherwise". But now I see that the default `-1` will set the same CPU count as I did here.

Reply:
So do we need to change anything?

Reply:
we can safely revert this. nothing will change.